### PR TITLE
[ci] Change conda install location

### PR DESCRIPTION
**Pull request by guolinke**

### CODE REVIEW

**Comment by guolinke** [comment]
for https://github.com/Microsoft/LightGBM/pull/1849#issuecomment-439854619

File: .ci/dockers/ubuntu-14.04/Dockerfile [hunk 1]
```diff
@@ -43,13 +43,15 @@ RUN curl -sL https://cmake.org/files/v3.12/cmake-3.12.0-Linux-x86_64.sh -o cmake
...
  && chmod +x miniconda.sh \
- && ./miniconda.sh -b -p /usr/local/miniconda \
+ && ./miniconda.sh -b -p /opt/conda \
  && rm miniconda.sh \
- && chmod -R 777 /usr/local/miniconda \
- && /usr/local/miniconda/bin/conda install python=3 -q -y \
- && /usr/local/miniconda/bin/conda install mkl qt -q -y \
- && /usr/local/miniconda/bin/conda clean -a -y
-ENV CONDA=/usr/local/miniconda
+ && chmod -R 777 /opt/conda \
```
Comment (StrikerRUS) [inline .ci/dockers/ubuntu-14.04/Dockerfile]: 1

File: .ci/dockers/ubuntu-14.04/Dockerfile [hunk 2]
```diff
@@ -43,13 +43,15 @@ RUN curl -sL https://cmake.org/files/v3.12/cmake-3.12.0-Linux-x86_64.sh -o cmake
...
+ && /opt/conda/bin/conda install python=3 -q -y \
+ && /opt/conda/bin/conda install mkl qt -q -y \
+ && /opt/conda/bin/conda clean -a -y \
+ && chmod -R 777 /opt/conda
```
Comment (StrikerRUS) [inline .ci/dockers/ubuntu-14.04/Dockerfile]: 2

**Comment by guolinke** [comment]
@StrikerRUS I just remove all chmod, you can try it after the dev image built. 
If that cannot work, you can try to add the last chmod.

**Comment by StrikerRUS** [comment]
@guolinke OK.

**Comment by StrikerRUS** [comment]
@guolinke `chmod` is needed, **Permission error** otherwise.

**Comment by StrikerRUS** [comment]
@guolinke I've checked - everything OK now. Please merge, because GitHub doesn't allow me to do it due to red CIs.

**Comment by chivee** [comment]
it seems the test still using `usr/local/miniconda` as envs.

**Comment by StrikerRUS** [comment]
Thanks @chivee for merging!

**Pull request merged**